include/macros.h

```diff
@@ -214,3 +214,11 @@ extern struct GraphicsContext* __gfxCtx;
     } while (0)
 
 #endif
+
+#ifdef DISABLE_LOOKUP_TABLE
+
+#define ONE 1.0f
```

Comment:
why using a macro for this? it looks weird to me 🤔 (the ``ONE`` one)

Reply:
yeah i thought i would need to do the function twice and forgot